### PR TITLE
athenacli 1.4.1 (new formula)

### LINKED ISSUE #59: Homebrew option
I'd love to install the `athenacli` using homebrew instead of pip so it doesn't clutter global dependencies

    brew install athenacli

### PR DESCRIPTION
**Pull request by chenrui333**
closes dbcli/athenacli#59

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/athena-cli.rb [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  desc "CLI tool for AWS Athena service"
+  homepage "https://athenacli.readthedocs.io/en/latest/"
+  url "https://files.pythonhosted.org/packages/7b/d8/8e3525213c209cb6de70ef9074fe356bada9e8871b6b85fcd860d571071b/athenacli-1.4.1.tar.gz"
+  sha256 "ad70993238d718a8eae5b573b12af83574ceb85ae694daa5c2e98215f89a5423"
+  license "BSD-3-Clause"
+
+  depends_on "python@3.8"
+
+  # Generated with homebrew-pypi-poet from:
+  # poet -r athenacli \
```
Comment (dawidd6) [inline Formula/athena-cli.rb:13]: `brew update-python-resources` doesn't work?

Comment (chenrui333) [reply Formula/athena-cli.rb:13]: oh, I did not try that route. I can remove this to avoid confusion for the future.

File: Formula/athenacli.rb [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  end
+
+  resource "wcwidth" do
+    url "https://files.pythonhosted.org/packages/89/38/459b727c381504f361832b9e5ace19966de1a235d73cdbdea91c771a1155/wcwidth-0.2.5.tar.gz"
+    sha256 "c4d647b99872929fdb7bdcaa4fbe7f01413ed3d98077df798530e5b04f116c83"
+  end
+
+  def install
+    virtualenv_install_with_resources
+    bin.install_symlink "athenacli" => "athena-cli"
```
Comment (dawidd6) [inline Formula/athenacli.rb:114]: It's that link necessary?

Comment (chenrui333) [reply Formula/athenacli.rb]: yeah, probably no as the formula name got changed, just fixed this symlink thing.

**Review by dawidd6 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @dawidd6!!

**Pull request closed without merging**